setup.py

```diff
@@ -25,4 +23,7 @@
         ],
     packages = ['isbn'],
     package_dir = {'isbn': 'isbn'},
-    )
+    autosemver={
+        'bugtracker_url': True,
```

Comment:
I actually think that pyroma there might give false positives, as if you use autosemver you don't need to pass the version keyword that otherwise it requires. Probably would have to add some custom rules to pick up the autosemver parameters too.